Next

Meteorite opened the pull request:
Fix getURI hanging on downloading big file with ReturnType.direct

Meteorite committed:
fix hanging on AllData with large file

Meteorite committed:
add Idea's stuff to .gitignore

Bombe commented on .gitignore:
```diff
@@ -1,3 +1,5 @@
 bin/
 doc/
 target/
+.idea/
```
This does not belong here but in your global .gitignore.

Bombe commented on src/test/java/net/pterodactylus/fcp/TempInputStreamTest.java:
```diff
@@ -0,0 +1,54 @@
+package net.pterodactylus.fcp;
```
This whole file uses CRLF as line separator and spaces instead of tabs. Do you want to fix that or should I do it?

Bombe commented:
Thank you for this pull request! I wish I had seen it before I released 0.1.5 yesterday. :) If you can address the mentioned issues I will merge/cherry-pick this before the next release.

Meteorite committed:
switch to LF and tabs in TestInputStreamTest.java, remove Idea's stuff from project's gitignore

Meteorite commented:
Ok, I've changed indentation and line endings to fit project's style. I've also removed .gitignore additions.

I personally prefer .gitignore and .gitattributes of each project to be as explicit as possible. It helps to decrease the chance for newbies to misfit project's style, like my misfit of line endings here :) It also makes everything less dependent on other guys setup. But, your project - your rules :)

Bombe commented:
This looks way better, merging it right now. Thank you very much!

(I prefer to put files specific to my IDE in my global .gitignore so that I don’t ever commit them to any project I work on. The only thing that belongs in a .gitignore is stuff that is created by the project’s build script itself. (Then again, the .gitignore in this project is very old and might actually contain stuff that I wouldn’t accept nowadays. Ah, well, what can you do.))
